doc/modules/cassandra/pages/new/virtualtables.adoc

```diff
@@ -99,6 +99,9 @@ recent_hit_rate_per_second, recent_request_rate_per_second, request_count, and s
 |thread_pools |Lists metrics for each thread pool.
 
 |tombstones_per_read |Records counts, keyspace_name, tablek_name, max, and median for tombstones.
+
+|cql_metrcs |Lists metrics for CQL prepared statements.
```

Comment:
this should be sorted. Can you move it up?

Reply:
The metrics also include `regular_statements_executed`. The description can be 
"Lists metrics for CQL statements."

Reply:
https://cassandra.apache.org/doc/latest/cassandra/operating/metrics.html#:~:text=org.apache.cassandra.metrics%3Atype%3DCQL,that%20are%20prepared%20vs%20unprepared doc mentions as cql prepared statements metric, so trying to be inline with this.

Reply:
> this should be sorted. Can you move it up?

done